tests/problem1Tests.rb

```diff
@@ -0,0 +1,37 @@
+require 'test/unit'
+require '../src/problem1'
+
+class Problem1Tests < Test::Unit::TestCase
+
+  def setup
+    @problem1 = Problem1.new
+    @randomIntegerTestData = [1,4,7,4,10,6]
+    @orderedIntegerTestData = (1..100).to_a
+    @oneIntegerArray = [6]
+    @oddNumbersArray = [1,3,5,7,1321]
+  end
+
+  def teardown
+
+  end
```

Comment:
Are you required to implement this method in the test case even if it does nothing ?

Reply:
Of course not, you are right, it can be scrapped :)